testapi.pm

```diff
@@ -971,11 +971,12 @@ sub assert_script_run {
             # not change default timeout.
             timeout      => 90,
             fail_message => '',
-            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS')
+            quiet        => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            type_speed => testapi::get_var('VNC_TYPING_LIMIT')
         }, ['timeout', 'fail_message'], @_);
 
     bmwqemu::log_call(cmd => $cmd, %args);
-    my $ret = $distri->script_run($cmd, timeout => $args{timeout}, quiet => $args{quiet});
+    my $ret = $distri->script_run($cmd, timeout => $args{timeout}, quiet => $args{quiet}, type_speed => $args{type_speed});
```

Comment:
I think, that we should use the same name to be consistent.

```suggestion
    my $ret = $distri->script_run($cmd, timeout => $args{timeout}, quiet => $args{quiet}, max_interval => $args{max_interval});
```

Reply:
I would prefer a speaking name for the user, max_interval could be anything but type_speed is clear in what it does